File: coding_test/baekjoon/q_10813.cpp
// 공 바꾸기
#include <stdio.h>

int main()
{
    int n, m;
    scanf("%d %d", &n, &m);
    
    int* n_arr = new int[n];
    for (int i = 0; i < n; i++)
        n_arr[i] = i + 1;

    for (int i = 0; i < m; i++)
    {
        int n1, n2;
        scanf("%d %d", &n1, &n2);
        int tmp = n_arr[n1 - 1];
        n_arr[n1 - 1] = n_arr[n2 - 1];
        n_arr[n2 - 1] = tmp;
    }

    for (int i = 0; i < n; i++)
        printf("%d ", n_arr[i]);

    delete[] n_arr;

    return 0;
}

File: coding_test/baekjoon/q_10813.py
# 공 바꾸기
n, m = map(int, input().split())
n_arr = [i for i in range(1, n + 1)]

for _ in range(m):
    i, j = map(int, input().split())
    tmp = n_arr[i - 1]
    n_arr[i - 1] = n_arr[j - 1]
    n_arr[j - 1] = tmp

print(" ".join(map(str, n_arr)))